flow/util/genMetrics.py

```diff
@@ -831,6 +831,7 @@ def extract_metrics(cwd, platform, design, flow_variant, output, hier_json):
 
     with open('metrics.html', 'w') as f:
         f.write(all_df.to_html())
+    all_df.to_csv('metrics.csv', index = False)
```

Comment:
Will this file be used anywhere?

Reply:
It is easy to copy the result from the .csv file to a google sheet instead of importing an HTML. So, I added this code to dump the .csv file.